test/suites/dev/moonbase/test-fees/__snapshots__/test-length-fees2.ts.snap

```diff
@@ -0,0 +1,3 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
```

Comment:
Are those files intended ?

Reply:
What is the benefit of the snapshot ?

Reply:
replace the file snapshots with inline snapshots